### PR TITLE
[node] Add process.release

### PR DESCRIPTION
**Pull request by fileformat**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://nodejs.org/api/process.html#process_process_release
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/node/index.d.ts [hunk 1]
```diff
@@ -647,6 +647,14 @@ declare namespace NodeJS {
         system: number;
     }
 
+    export interface ProcessRelease {
+        name: string;
+        sourceUrl: string;
+        headersUrl: string;
+        libUrl: string;
+        lts: string;
```
Comment (Flarna) [inline types/node/index.d.ts]: `lts` should be `string | undefined` according to docs.

**Review by Flarna — APPROVED** [review]

**Review by r3nya — COMMENTED** [review]
Hey! Can you add it into `node/v8/index.d.ts` as well please?

**Comment by r3nya** [comment]
@fileformat yeah, you're right. Please take a look – https://nodejs.org/dist/latest-v8.x/docs/api/process.html#process_process_release

**Comment by SimonSchick** [comment]
Tests?

File: types/node/index.d.ts [hunk 2]
```diff
@@ -647,6 +647,14 @@ declare namespace NodeJS {
         system: number;
     }
 
+    export interface ProcessRelease {
+        name: string;
+        sourceUrl: string;
+        headersUrl: string;
+        libUrl: string;
```
Comment (SimonSchick) [inline types/node/index.d.ts]: This should be `libUrl?: string` also generally use the `?` over `undefined`.

**Review by SimonSchick — CHANGES_REQUESTED** [review]

File: types/node/index.d.ts (hunk 2 above)
Comment (isLishude) [reply types/node/index.d.ts]: > libUrl is only present on Windows builds of Node.js and will be missing on all other platforms.

**Review by islishude — CHANGES_REQUESTED** [review]

File: types/node/index.d.ts (hunk 2 above)
Comment (isLishude) [reply types/node/index.d.ts]: In custom builds from non-release versions of the source tree, only the > name property may be present. The additional properties should not be relied upon to exist.

**Review by islishude — CHANGES_REQUESTED** [review]

**Comment by fileformat** [comment]
- changes made based on the comments
- added to v8

**Review by SimonSchick — APPROVED** [review]

**Review by r3nya — APPROVED** [review]

**Review by islishude — APPROVED** [review]

**Pull request merged**